package.json

```diff
@@ -26,20 +26,21 @@
     "create-package": "node ./scripts/package-builder.js",
     "pre-build": "node ./scripts/pre-build.js",
     "build": "npm-run-all --parallel build:ts \"build:packages {@}\" --",
-    "build:packages": "./scripts/tsnode ./scripts/build.ts",
+    "build:packages": "yarn ts-node ./scripts/build.ts",
     "build:ts": "tsc --build build.tsconfig.json",
     "build:storybook": "build-storybook",
     "check": "npm-run-all --parallel check:*",
     "check:prettier": "prettier --check \"**/*.{js,ts,tsx,json,md}\"",
     "check:eslint": "eslint \"**/*.{js,ts,tsx}\"",
     "check:deps": "ts-node -O '{\"module\": \"commonjs\", \"resolveJsonModule\": true}' ./scripts/depcheck.ts",
+    "check:builds": "yarn ts-node ./scripts/check-builds.ts",
     "check:ts": "tsc --project tsconfig.json",
     "clean": "yarn clean:builds && yarn clean:modules",
-    "clean:builds": "rm -rf packages/*/dist packages/*/tsconfig.tsbuildinfo",
-    "clean:modules": "rm -rf packages/*/node_modules node_modules",
+    "clean:builds": "rm -rf packages/*/dist packages/*/tsconfig.tsbuildinfo website/tsconfig.tsbuildinfo ./tsconfig.tsbuildinfo",
```

Comment:
I assume this website/... will go away pretty soon?

Reply:
yeah 👍